src/JBMultiTerminal.sol

```diff
@@ -1696,6 +1699,11 @@ contract JBMultiTerminal is JBPermissioned, ERC2771Context, IJBMultiTerminal {
 
                 // Add balance back to the project.
                 _recordAddedBalanceFor(projectId, token, leftoverPayoutAmount);
+
+                // If the project owner is not feeless, subtract the leftover payout amount from the amount eligible for fees.
+                if (!isFeeless) {
+                    amountEligibleForFees -= leftoverPayoutAmount;
```

Comment:
There is little discrepancy, because `leftoverPayoutAmount` here is with the deducted fee, but when we make 
`amountEligibleForFees += leftoverPayoutAmount;`, it is the original amount

Example:
If `leftoverPayoutAmount == 1000` and `feePercent = 5%`, `amountEligibleForFees = 500`
- new `amountEligibleForFees`=  500 + 1000 = 1500
- new `leftoverPayoutAmount` = 1000 - 50 = 950
- return 950 to store in the catch block
- new `amountEligibleForFees` = 1500 - 950 = 550, instead of 500

Reply:
nice catch, again. fixed! check it out